modules/nodes-application-secrets-creating.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-pods-secrets.adoc
+
+:_content-type: PROCEDURE
+[id="nodes-application-secrets-creating-sa_{context}"]
+= Creating a service account token secret for applications
+
+As an administrator, you can create a service account token secret, which allows you to distribute a service account token to applications that must authenticate to the API.
+
+.Procedure
+
+. Create a service account in your namespace.
```

Comment:
@sunilcio Could you please suggest the method for creating a service account in the namespace? Is it a command?